### DISCUSSION TITLE
Project name

### DISCUSSION THREAD

**Discussion by solarkraft** [fablab-luenen/dokuwiki-preconfigured#34] [General]
This distribution is supposed to be a bit more than the current name suggests. Let's find a fitting name.

**Comment by solarkraft** [comment]
![image](https://user-images.githubusercontent.com/1337470/131467883-cc1d60db-a99f-4ebc-95ae-e78b685b6799.png)
dookuwiki

**Comment by solarkraft** [comment]
![A8B1A9BE-8BC4-4F37-BF0A-1456DEC034D4](https://user-images.githubusercontent.com/1337470/132998227-ac0c013e-ce9d-4930-be0a-ebb8562e8524.jpeg)
Dogekuwiki

**Comment by bruciexx** [comment]
based
